### DISCUSSION TITLE
Validation of SimNIBS' FEM simulations

### DISCUSSION THREAD

**Discussion by grodgers1** [simnibs/simnibs#284] [2. General]
Hi SimNIBS team, thanks for the great tool! 

Have SimNIBS' FEM calculations validated or compared to other tools such as commercial software? Specifically I am interested in TES simulations.

In terms of validation of SimNIBS, I am aware of the very nice paper by G. B. Saturnino et al., J. Neural Eng. 2019 (10.1088/1741-2552/ab41ba) comparing SimNIBS with analytical solutions for spheres and SimNIBS 3.0 with SimNIBS 2.1 for a head model. 

Are there anything else out there that I am missing?

Thanks a lot!

**Comment by axelthi** [comment]
Hi, the Saturnino et al paper also contains convergence tests, which are an important part of the validation, in addition to the comparisons with the analytical solutions.There is also this comparison: https://ieeexplore.ieee.org/document/9175802 that shows an around 5% difference in the gray matter area with the highest fields between our FEM and BEM-FMM. Taken together, the SimNIBS implementation performs as expected for a 1st order FEM with SPR. It's relevant to realize that numerical errors are only one of several factors that contribute to the differences of the simulated fields to the (unknown) true fields.  We have set the standard mesh density for the head models so that the numerical errors in gray matter are low compared to the the likely impact of segmentation inaccuracies (discussed in Saturnino et al) or not well known tissue conductivities (10.1016/j.neuroimage.2018.12.053) on the simulated fields in gray matter. If needed, you can always make the head meshes denser to improve numerical accuracy further, but this is only relevant for specific aims.

**Reply by grodgers1** [reply]
Great, thanks a lot for your explanation, that makes sense -- segmentation and conductivity errors will have a bigger impact than the numerical errors.
